src/ocaml_lsp.ml

```diff
@@ -1,4 +1,40 @@
 open! Import
+open Interop
+
+module OcamllspSettingEnable = struct
+  include Interface.Make ()
+
+  include
+    [%js:
+    val enable : t -> bool or_undefined [@@js.get]
+
+    val create : enable:bool -> unit -> t [@@js.builder]]
+
+  let create ~enable = create ~enable ()
```

Comment:
The unit parameter is not necessary for `js.builder` functions: https://github.com/LexiFi/gen_js_api/blob/master/VALUES.md

That parameter can just be removed instead of adding a wrapper function:

```ocaml
val create : enable:bool -> t [@@js.builder]
```

Reply:
Thanks. Fixed it.